diff --git a/lib/Utils/logger.d.ts b/lib/Utils/logger.d.ts
index 4d78996..059f58f 100644
--- a/lib/Utils/logger.d.ts
+++ b/lib/Utils/logger.d.ts
@@ -7,6 +7,6 @@ export interface ILogger {
     info(obj: unknown, msg?: string): void;
     warn(obj: unknown, msg?: string): void;
     error(obj: unknown, msg?: string): void;
 }
-declare const _default: import("pino").Logger<never, boolean>;
+declare const _default: ILogger;
 export default _default;
